4-storage/storage-loader/lib/snowplow-storage-loader/redshift_loader.rb

```diff
@@ -133,18 +133,21 @@ def get_shredded_statements(config, target, s3)
           schema = extract_schema(target[:table])
 
           ShreddedType.discover_shredded_types(s3, config[:aws][:s3][:buckets][:shredded][:good], schema).map { |st|
-
-            jsonpaths_file = st.discover_jsonpaths_file(s3, config[:aws][:s3][:buckets][:jsonpath_assets])
-            if jsonpaths_file.nil?
-              raise DatabaseLoadError, "Cannot find JSON Paths file to load #{st.s3_objectpath} into #{st.table}"
+            if target[:shredded_types][:exclude].include?(st.table)
```

Comment:
It looks like we are checking the list of shredded types against the table name, rather than the type name?